pkg/controllers/scheduling/clustersetbinding_event_handler.go

```diff
@@ -0,0 +1,68 @@
+package scheduling
+
+import (
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/klog/v2"
+
+	clusterlisterv1alpha1 "github.com/open-cluster-management/api/client/cluster/listers/cluster/v1alpha1"
+	clusterapiv1alpha1 "github.com/open-cluster-management/api/cluster/v1alpha1"
+)
+
+type clusterSetBindingEventHandler struct {
+	clusterSetLister     clusterlisterv1alpha1.ManagedClusterSetLister
+	placementLister      clusterlisterv1alpha1.PlacementLister
+	enqueuePlacementFunc enqueuePlacementFunc
+}
+
+func (h *clusterSetBindingEventHandler) OnAdd(obj interface{}) {
+	h.onChange(obj)
+}
+
+func (h *clusterSetBindingEventHandler) OnUpdate(oldObj, newObj interface{}) {
+	h.onChange(newObj)
+}
+
+func (h *clusterSetBindingEventHandler) OnDelete(obj interface{}) {
+	h.onChange(obj)
```

Comment:
delete has to handle partial object  meta.

Reply:
Done.